components/ai_chat/resources/page/components/conversation_list/style.module.scss

```diff
@@ -4,45 +4,102 @@
 // you can obtain one at https://mozilla.org/MPL/2.0/.
 
 .list {
-  padding: 16px;
-}
-
-.textBase {
-  p {
-   margin: 0;
-   white-space: pre-wrap;
+  div[data-turn='assistant'] {
+    padding-bottom: 0;
   }
- }
+}
 
 .turnAI {
-  @extend .textBase;
+  display: flex;
+  align-items: flex-start;
+  gap: 16px;
+  font: var(--leo-font-secondary-default-regular);
+  color: var(--leo-color-text-primary);
   padding: 16px;
 }
 
 .turnHuman {
-  @extend .textBase;
-  padding: 0 16px 16px 0;
-
-  // bubble wrap
-  p:first-child {
-    background-color: var(--leo-color-container-highlight);
-    padding: 16px;
-    border-radius: 16px 16px 2px 16px;
-  }
+  display: flex;
+  align-items: flex-start;
+  gap: 16px;
+  font: var(--leo-font-secondary-default-regular);
+  color: var(--leo-color-text-primary);
+  padding: 16px;
+
+  background-color: var(--leo-color-page-background);
+  margin-top: 16px;
 }
 
 .caret {
-  @extend .textBase;
   display: inline-block;
   margin-left: 5px;
   width: 8px;
   height: 12px;
   background-color: currentColor;
 }
 
-.suggestedQuestions {
+.suggestedQuestionsBox {
+  padding: 16px 16px 16px 64px;
+
+  // TODO(nullhook): Make this change in dev leo
+  leo-button {
+    span {
+      align-items: center;
+    }
+  }
+
+  div {
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    gap: 8px;
+  }
+}
+
+.suggestedQuestionLabel {
+  font: var(--leo-font-secondary-small-regular);
+  color: var(--leo-color-text-secondary);
+  padding-bottom: 16px;
+}
+
+.avatarHuman {
+  --leo-icon-size: 18px;
+
+  width: 32px;
+  height: 32px;
+  background-color: var(--leo-color-container-background);
+  border-radius: 100px;
+  flex-shrink: 0;
+  color: var(--leo-color-icon-default);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.avatarAI {
+  --leo-icon-size: 18px;
+
+  width: 32px;
+  height: 32px;
+  border-radius: 100px;
+  flex-shrink: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: linear-gradient(314.42deg, #FA7250 8.49%, #FF1893 43.72%, #A78AFF 99.51%);
+  color: white;
+}
+
```

Comment:
Also a lot of duplication here, seems like we can combine the rules

Reply:
added new class, removed dupe